src/NUnitFramework/testdata/TestCaseOrderAttributeFixture.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NUnit.Framework;
+
+namespace NUnit.TestData
+{
+    [TestFixture]
+    public class TestCaseOrderAttributeFixture
+    {
+        [Test]
+        [Order(3)]
+        public void Z_ThirdTestWithSameOrderAsSecond()
```

Comment:
You say third test with same order as second, did you intend this to have `Order(2)`?